src/lib/db.js

```diff
@@ -684,8 +689,12 @@ export const DB = {
     if (!id) return Message.error('Unable to delete relic')
     DB.unequipRelicById(id)
     const relicsById = window.store.getState().relicsById
+    const ageIndex = relicsById[id].ageIndex
     delete relicsById[id]
     window.store.getState().setRelicsById(relicsById)
+    DB.getRelics()// update assuming the delete operation is matched in game
+      .filter((x) => x.ageIndex > ageIndex)
+      .forEach((x) => x.ageIndex--)
```

Comment:
I'm not sure if this is a good assumption, since ingame we can just mark it with the Discard icon